sriov/common/utils.py

```diff
@@ -653,3 +653,40 @@ def execute_until_timeout(ssh_obj: ShellHandler, cmd: str, timeout: int = 10) ->
         time.sleep(1)
     print("\nstdout:" + str(out) + "\nstderr:" + str(err))
     return False
+
+
+def wait_tmux_testpmd_ready(ssh_obj: ShellHandler, tmux_session: str,
+                            timeout: int) -> bool:
+    """wait the testpmd in a tmux session ready
```

Comment:
Missing a word (either "until" or "for"), for example:"Wait until the testpmd in a tmux session is ready"

Reply:
done